src/main/java/com/strandls/naksha/pojo/LayerAttributes.java

```diff
@@ -1,11 +1,5 @@
 package com.strandls.naksha.pojo;
 
-/**
- * Attributes present in a layer
- * 
- * @author mukund
- *
- */
```

Comment:
please don't remove credits and remove this kinda changes so there can be clean diff